src/App.tsx

```diff
@@ -17,18 +50,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectedOption={selectedOption}
+                query={query}
+                setSelectedOption={setSelectedOption}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {
+                !loader
+                  ? <Loader />
```

Comment:
```suggestion
              {isLoading
                  ? <Loader />
```
you don't need `!` here! you allready get boolean